setup/scripts/temporal_manager.sh

```diff
@@ -13,33 +13,43 @@ export CONFIG_DAG
 case "$1" in
 
     api)
+        INIT_DB=yes
+        export INIT_DB
         temporal api
         ;;
     queue-dfa)
+        INIT_DB=yes
```

Comment:
hm why not `true` instead of `yes` (since `true` as env toggles is already used elsewhere in the codebase)

Reply:
Done